examples/compute/region-disk.gyro

```diff
@@ -0,0 +1,33 @@
+google::compute-region-disk region-disk-example
+    name: "region-disk-example"
+    description: "region-disk-example-desc"
+    region: "us-central1"
+
+    replica-zones: [
+         "us-central1-c",
+         "us-central1-a"
+    ]
+
+    size-gb: 32
+    type: "pd-ssd"
+    labels: {
+        label-key: 'label-value'
+    }
+    physical-block-size-bytes: 16384
+end
+
+google::compute-region-disk region-disk-snapshot-example
+    name: "region-disk-snapshot-example"
+    region: "us-central1"
+
+    replica-zones: [
+         "us-central1-c",
+         "us-central1-a"
+    ]
+
+    source-snapshot: $(google::compute-snapshot region-snapshot-example)
```

Comment:
Have the snapshot resource example in this gyro config for this disk config to be valid